Src/Plugins/Security/SEC101_038.PostgreSqlCredentialsValidator.cs

```diff
@@ -61,6 +64,16 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
             groups.TryGetNonEmptyValue("resource", out FlexMatch resource);
 
             string hostValue = FilteringHelpers.StandardizeLocalhostName(host.Value);
+            string idValue = id.Value;
+
+            // Username must be in the form <username>@<hostname> to communicate with Azure.
+            // If the username does not contain a host name, we can't connect.
+            if (hostValue.Contains(AzureHost) &&
+                              !idValue.Contains("@"))
```

Comment:
here u can keep everything in one line.
Our idea is:
- the length should not be too long
- the length should be ordered  by length (check like 56-58).

Reply:
fixed